### PR TITLE
Scratch/cta section

### PR DESCRIPTION
**Pull request by Bibhuti05**
image

image

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -27,6 +28,9 @@ export default function Home() {
   const refCarouselUpcomingSubscription = useRef<HTMLDivElement>(null);
   const isVisibleCarouselUpcomingSubscription = useOnScreen(refCarouselUpcomingSubscription, '70px');
 
+  const refCTABox = useRef<HTMLDivElement>(null);
+  const isVisibleCTABox = useOnScreen(refCTABox, '0px');
```
Comment (arijeet-mukherjee) [inline src/app/page.tsx:32]: increase this root margin

Comment (Bibhuti05) [reply src/app/page.tsx]: increased

File: src/component/CTABox/index.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import CardBox from '@component/cardBox'
+import PlansCard from './PlansCard';
```
Comment (arijeet-mukherjee) [inline src/component/CTABox/index.tsx:3]: use dynamic import here

File: src/component/CTABox/index.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import CardBox from '@component/cardBox'
```
Comment (arijeet-mukherjee) [inline src/component/CTABox/index.tsx:2]: here too use dynamic import

File: src/component/CTABox/index.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    goTo,
+    paddingLeftContent,
+    inputBox,
+    bulletPointImg,
+    bulletPoints,
+    childProps }) => {
+
+    return (
+        <CardBox
+
```
Comment (arijeet-mukherjee) [inline src/component/CTABox/index.tsx:52]: use fullGradient true for mobile device

**Comment by Bibhuti05** [comment]
Convoluted the PR with unecessary changes 
https://github.com/arijeet-mukherjee/app-home-frontend/pull/25

**Pull request closed without merging**